api/base/urls.py

```diff
@@ -23,6 +24,7 @@
                 url(r'^chronos/', include('api.chronos.urls', namespace='chronos')),
                 url(r'^meetings/', include('api.meetings.urls', namespace='meetings')),
                 url(r'^metrics/', include('api.metrics.urls', namespace='metrics')),
+                url(r'^registries/(?P<provider_id>\w+)/bulk_create/(?P<filename>.*)/$', RegistrationBulkCreate.as_view(), name='bulk_create_csv'),
```

Comment:
It's currently here because it's an `_/` route. We should probably come up with a paradigm for "private" routes that share much of their path with "public" routes; that may end up being something like an `_urls.py` file and `url(r'^registries/', include('api.registries._urls'), namespace='registries')` here, but unsure if namespace collisions would be a problem.

I wouldn't necessarily consider this a blocker, but is a `TODO`

Reply:
The original plan was for this the be under the registration provider routes. But if we are to include this as a private route under `/_` namespace, we'd have to do it here.